src/tuntap/interface.cr

```diff
@@ -50,16 +50,24 @@ module Tuntap
     # Adds an IP address to the interface.  For this to work, the interface has
     # to be up.
     def add_address(address : String) : Nil
+      add_address(sockaddr(address))
+    end
+
+    def add_address(address : LibC::Sockaddr) : Nil
```

Comment:
Could you add a comment just saying `dito`? That should instruct the docs generator to copy the documentation from the method "above" verbatim.

Reply:
Example usage: https://github.com/crystal-lang/crystal/blob/25de751e005d509192f46228ff6e5c4c8b3e424e/src/csv/builder.cr#L71